src/main/js/ephox/imagetools/transformations/ImageTools.js

```diff
@@ -80,13 +82,13 @@ define("ephox/imagetools/transformations/ImageTools", [
 
   function resize(blob, w, h) {
     return Conversions.blobToImage(blob).then(function(image) {
-      var canvas = Canvas.create(w, h),
-        context = Canvas.get2dContext(canvas);
-
-      context.drawImage(image, 0, 0, w, h);
-      revokeImageUrl(image);
-
-      return Conversions.canvasToBlob(canvas, blob.type);
+      return ImageResizerWebgl.scale(image, w, h)
```

Comment:
While I agree that potentially various resizing algorithms should be exposed externally, but for now we had a goal of achieving better quality resizing. Both implementations at the moment achieve the similar result, so it was logical to have the fallback in place.

If we expose separate methods for WebGL and Canvas resizing (which are separate now) then try/catch for WebGL should still happen somewhere outside.